### DISCUSSION TITLE
RN tvos 67.4 release

### DISCUSSION THREAD

**Discussion by sureshsu** [react-native-tvos/react-native-tvos#545] [General Discussions]
Is there a possibility to have a React native tvos release for RN 67.4?

We are using React native for other targets with react native 67.4 and we are not going to migrate soon to a different version. Hence we want a react native tvos release for Apple TV with 67.4 compatibility

Appreciate if community can share a release of it.

Thanks

**Comment by douglowder** [comment]
This is not possible. We did publish 0.66 and 0.68 releases, but not 0.67.
